public/js/lib/themes/variables.css

```diff
@@ -20,7 +20,9 @@
   --theme-contrast-background: #e6b064;
 
   --theme-tab-toolbar-background: #fcfcfc;
+  --theme-faded-tab-background: #7E7E7E;
   --theme-toolbar-background: #fcfcfc;
+  --theme-toolbar-background-alt: #f5f5f5;
```

Comment:
would it be possible to commit these changes to gecko as well so the changes are consistent.